sdk/src/test/java/co/decodable/sdk/pipeline/internal/config/metadata/MetadataProcessorTest.java

```diff
@@ -110,10 +145,14 @@ var record = handler.getRecords().get(0);
     assertThat(record.getMessage()).contains("Neither source nor sink streams were declared");
   }
 
-  private static Compilation compile(URL fileWithoutAnnotations) {
+  private static Compilation compile(URL... files) {
+    Arrays.stream(files).map(f -> JavaFileObjects.forResource(f)).collect(Collectors.toSet());
```

Comment:
This line seems redundant

Reply:
How odd, I guess some left-over from experimentation with the streams API. Good catch!